### DISCUSSION TITLE
Retrieving User Profile Image (avatar)

### DISCUSSION THREAD

**Discussion by xenastro** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#94] [Q&A]
Hello,

Is there an endpoint in the WhatsApp Cloud API for retrieving user's profile images? I searched for it but couldn't find it.

I found an api to retrieve the business's profile image, but not the users, I mean people who converse with us over WhatsApp.

**Comment by gabrieldwight** [comment]
Hi, there is no endpoint for retrieving a user profile image in cloud API since it is unavailable.
